src/blog/2022/10/db-migraton-01.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Scheduled maintenance: Database encryption October 2022"
+subtitle: Moving to encrypted PostgreSQL storage
+description: Details on upcoming maintenance period
+date: 2022-10-13
+authors: ["ben-hardill"]
+---
+
+# Scheduled maintenance: Database encryption October 2022
+
+As part of an on going security [review](https://flowforge.com/product/security/#data-at-rest) of the FlowForge Managed offering we discovered that the backend database was not using encrypted storage. In keeping with industry best practices we plan to migrate the database to a new instance using encrypted at rest storage.
+
+## Impact
+
+Customers' Node-RED instance will remain running, though any features that depend on FlowForge will not operate as expected. This includes user sessions, project and team management, as well as the project nodes for inter-project communication.
+
+Self hosted instalations are uneffected by this change.
+
+## When
+
+The plan is to carry out this work at XX October 2022 at XX:XX UTC and the work should be complete in 2hr with the system being restored earlier if all is well.
```

Comment:
```suggestion
The migration will be on XX October 2022 at XX:XX UTC and is expected to take under 2 hours. The platform will be available as soon as the migration is complete.

We will post updates during the migration period to our Twitter account [https://twitter.com/flowforgeinc](@FlowForgeInc).
```

Reply:
Love the update part on twitter, thanks for the suggestion.